source/server/listener_manager_impl.h

```diff
@@ -112,6 +112,7 @@ using ListenerImplPtr = std::unique_ptr<ListenerImpl>;
   COUNTER(listener_added)                                                                          \
   COUNTER(listener_create_failure)                                                                 \
   COUNTER(listener_create_success)                                                                 \
+  COUNTER(listener_in_place_updated)                                                               \
```

Comment:
This needs documenting?

Reply:
Also please check this in an integration test.

Reply:
oh, you mean stats.rst. Get it.
Aha, yes, it's last minute change and it's not reflected in integration test